packages/angular/cli/lib/config/workspace-schema.json

```diff
@@ -389,6 +389,28 @@
               "additionalProperties": false,
               "required": ["builder"]
             },
+            {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "builder": {
+                  "const": "@angular/build:application"
```

Comment:
You need to add the new builder in https://github.com/angular/angular-cli/blob/b50ad71503af3c9d34be596686206b3e16450b32/packages/angular/cli/lib/config/workspace-schema.json#L357